plugins/pencilblue/controllers/actions/user/manage_account/profile.js

```diff
@@ -61,9 +61,15 @@ module.exports = function UserProfileApiControllerModule(pb) {
      * @param {Function} cb
      */
     UserProfileApiController.prototype.put = function(cb) {
+        var self = this;
         var dto = this.body || {};
-        dto[pb.DAO.getIdField()] = this.session.authentication.user_id;
-        this.service.save(dto, this.handleSave(cb, false));
+        var sessionUserId = this.session.authentication.user_id;
+
+        this.service.getById(sessionUserId, function(err, value) {
+            dto.admin = value.admin;
```

Comment:
If there is an error, you need to set this value to the lowest possible. Please check for err here